app/views/provider_interface/application_choices/show.html.erb

```diff
@@ -14,7 +14,7 @@
 <% end %>
 
 <h1 class="govuk-heading-xl">
-  <span class="govuk-caption-xl">Application for <%= @application_choice.course.name_and_code %></span>
+  <span class="govuk-caption-xl">Application for <%= @application_choice.provider.name_and_code %> - <%= @application_choice.course.name_and_code %></span>
```

Comment:
If this title were a component then it would be possible to share the "only show provider name when user has access to multiple providers" logic here and also take some logic out of that view, which might be nice?

Reply:
I'm inclined to keep it like it is, to avoid having many permutations of the application page. Things become simpler if all application choice pages look the same.